### PR TITLE
Annihilates the global iterator

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
This PR removes the global iterator, and oh boy is this long overdue

The global iterator is basically the wish.com clone of object processing. Instead of everything being handled in one place, its handled on a per object level. And it has no tick checking. TLDR: Its shit

## Why It's Good For The Game
We shouldnt have awful code still in use, when we have much better things (StonedMC and SSobj) to use it with

## Changelog
:cl: AffectedArc07
del: Removes global iterators. Rejoice!
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Don't worry about the CL--players don't know what the heck a global iterator is.

**Comment by AffectedArc07** [comment]
I do it for the sake of completeness

**Comment by Spacemanspark** [comment]
Hey don't knock aliexpress I bought a 50$ phone there once and it's actually really decent

**Comment by AffectedArc07** [comment]
They do sell good LED panels, ill give them that. I'll change it to wish.com.

File: code/modules/spacepods/spacepod.dm [hunk 1]
```diff
@@ -127,6 +123,7 @@
 	cargo_hold.storage_slots = 0	//You need to install cargo modules to use it.
 	cargo_hold.max_w_class = 5		//fit almost anything
 	cargo_hold.max_combined_w_class = 0 //you can optimize your stash with larger items
+	START_PROCESSING(SSobj, src)
```
Comment (Fox-McCloud) [inline code/modules/spacepods/spacepod.dm:126]: Need to remove the `START_PROCESSING(SSobj, src)` from `/obj/spacepod/emp_act(severity)`

Comment (AffectedArc07) [reply code/modules/spacepods/spacepod.dm:126]: Done

**Pull request merged**